osv/ecosystems/_ecosystems.py

```diff
@@ -65,8 +65,10 @@
     'ConanCenter': OrderingUnsupportedEcosystem(),
     'GitHub Actions': OrderingUnsupportedEcosystem(),
     'Linux': OrderingUnsupportedEcosystem(),
+    'openSUSE': OrderingUnsupportedEcosystem(),
     'OSS-Fuzz': OrderingUnsupportedEcosystem(),
     'Photon OS': OrderingUnsupportedEcosystem(),
+    'SUSE': OrderingUnsupportedEcosystem(),
 }
 
 # Semver-based ecosystems, should correspond to _ecosystems above.
```

Comment:
Do we need to modify get() as well?

https://github.com/google/osv.dev/blob/7b948b1d877be849dfb65626caf85e59ac964b28/osv/ecosystems/_ecosystems.py#L111

Also this map looks out of date. E.g. Rocky Linux should be returning `RockyLinux` instead? 

For the actual SuSE ecosystem, I think we can just go ahead and add it properly now since it just uses RPM ordering under the hood.

Reply:
Updated the map and added sorting for SUSE.